packages/shared/src/components/post/write/CreatePostButton.tsx

```diff
@@ -16,7 +16,7 @@ export function CreatePostButton({
 }: CreatePostButtonProps): ReactElement {
   const { user, isAuthReady, squads } = useAuthContext();
   const { route, query } = useRouter();
-  const isTablet = useViewSize(ViewSize.Tablet);
+  const isMediumSize = useViewSize(ViewSize.Laptop);
```

Comment:
This should only fire on Laptop and higher?
Maybe keep the naming to `isLaptop` in that case to be consistent in the app.

Although pretty sure tablet is a reverse one and targets laptop and higher already.

Reply:
This logic was completely opposite to what is in Figma -> show small size button on screens of smaller than laptop size (based on my understanding of the designs). I can change the variable name 👍